h2o-algos/src/main/java/hex/schemas/GLMModelV3.java

```diff
@@ -113,9 +143,26 @@ public int compare(Integer o1, Integer o2) {
             tdt.set(i + 1, c, beta[i]);
         }
         coefficients_table.fillFromImpl(tdt);
+        if (n>2) {  // restore column names from pythonized ones
+          coefficients_table_multinomials_with_class_names.fillFromImpl(tdt);
+          revertCoeffNames(cols2, n, coefficients_table_multinomials_with_class_names);
+        }
       }
+
       return this;
     }
+
+    public void revertCoeffNames(String[] colNames, int nclass, TwoDimTableV3 coeffs_table) {
+      String newName = coeffs_table.name+" with class names";
+      coeffs_table.name = newName;
+      boolean bothCoeffStd = colNames.length==(2*nclass);
+      for (int tableIndex = 1; tableIndex <= nclass;  tableIndex++) {
```

Comment:
Why do we iterate from 1 to nclass and then do `tableIndex - 1 at each line of occurance ? Wouldn't it be better to just iterate from 0 ?

Reply:
Pavel:  The reason for the tableIndex starting from 1 is because for the coeffs_table.columns array, the first name is always "name".  The only useful assignment starts from index 1 to nclass.  So, if I set tableIndex to start from 1, I will need to do coeffs_table.columns[tableIndex+1].  Either way, I have to manipulate the tableIndex to either +1 or -1.  Since we are not looking for speed here, I just pick to do the tableIndex to start from 1.  Good observation!